tools/bootstrap/README.md

```diff
@@ -25,6 +25,16 @@ Install-BoxstarterPackage https://raw.githubusercontent.com/nodejs/node/master/t
 
 Entire installation will take up about 10 GB of disk space.
 
+### Why install two different versions of Python?
+Python 2 will reach its _end-of-life_ at the end of 2019. Afterwards, the
+interpreter will not get updates -- no bugfixes, no security fixes, nothing. In
```

Comment:
```suggestion
interpreter will not get updates — no bugfixes, no security fixes, nothing. In
```

Reply:
Why?

Reply:
`--` seems a bit hackish punctuation, it is rarely used in the docs as an m-dash replacement)